srcpkgs/dtools/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'dtools'
+pkgname=dtools
+# keep this synchronized with libphobos and dmd
+version=2.081.1
+revision=1
+build_wrksrc="tools-${version}"
+create_wrksrc=yes
+hostmakedepends="dmd"
+depends="libphobos>=${version}"
+short_desc="Ancillary tools for the D programming language"
+maintainer="Auri <me@aurieh.me>"
+license="Boost Software License 1.0"
+homepage="http://www.digitalmars.com/d/2.0/"
+distfiles="https://github.com/dlang/tools/archive/v${version}.tar.gz>dtools-${version}.tar.gz"
+checksum=71fa249dbfd278eec2b95ce577af32e623e44caf0d993905ddc189e3beec21d0
+
+do_build() {
+	# rdmd can't be built normally, is used to build others
+	dmd -od. -ofdtools-rdmd rdmd.d
+
+	for tool in DustMite/dustmite ddemangle catdoc detab tolf dget; do
```

Comment:
seems the Makefile builds more than just these [see here](https://github.com/dlang/tools/blob/master/posix.mak#L45). Any reason not to just use `build_style=gnu-makefile` and `make_build_args="-f posix.mak all"`?

Reply:
doing this builds `dmd` in `../dmd`, it can't use system `dmd` for some reason

Reply:
good call though, I seem to have missed of the 3 tools